docs/scripting/starting-scripts.md

```diff
@@ -0,0 +1,186 @@
+	
+	All API modules uses `[ 0, 1, 0 ]`. GameTest Framework experiment must be activated to use in world.
+
+-   1.19.20+
+	
+	All API modules have their versions incremented to `"1.0.0-beta"`, new features/changes are available in the beta version, and `0.1.0` has old developments and features in the API. GameTest Framework experiment must be activated to use in world.
+
+-   1.19.50+
+	
+	**Stable API modules**, these do not require the Beta APIs experiment to be turned on. The initial API set is narrow, but more features will be added over the coming months.
+
+	- @minecraft/server:
+		- `1.0.0`
+
+	**Beta API modules**, requires the Beta APIs experiment to be turned on in world settings, with lots of experimental features in the API.
+	
+	- @minecraft/server:
+		- `1.1.0-beta`
+		- `0.1.0` (use `mojang-minecraft`, NOT `@minecraft/server` when importing module for this version)
+
+	- @minecraft/server-ui:
+		- `1.0.0-beta`
+		- `0.1.0` (use `mojang-minecraft-ui`, NOT `@minecraft/server-ui` when importing module for this version)
+		
+	- @minecraft/server-gametest:
+		- `1.0.0-beta`
+
+	- @minecraft/server-net:
+		- `1.0.0-beta` (Bedrock dedidcated server module only, must enable in `permission.json`)
+
+	- @minecraft/server-admin:
+		- `1.0.0-beta` (Bedrock dedidcated server module only)
+
+As of release 1.19.30, you can also specify dependencies using the `module_name`:
+
+::: warning
+Do not add `"uuid"` and `"module_name"` properties into dependency, add one or the other.
+:::
+
+<CodeHeader>BP/manifest.json#dependencies[0]</CodeHeader>
+
+```json
+{
+	"module_name": "@minecraft/server",
+	"version": "1.1.0-beta"
+}
+```
+
+(Beta 1.19.40.23+) In order to use the `eval()` function or `Function()` constructors, you can add the following in the manifest capabilities:
+
+<CodeHeader>BP/manifest.json</CodeHeader>
+
+```json
+{
+	"capabilities": [
+		"script_eval"
+	]
+}
+```
+
+The entry point contains scripts and/or imports to other script files.
+
+## Writing scripts with JS
+
+Minecraft's scripting engine only runs JavaScript only, like any other JavaScript projects. Check out [Scripting with TypeScript](./typescript#script-api) for compiling TS directly to JavaScript.
+
+<CodeHeader>BP/scripts/Main.js</CodeHeader>
+
+```js
+// This file demonstrates that the code is working by
+// Spamming the chat with "Hello World"
+
+// Import world component from "@minecraft/server"
+import { world } from '@minecraft/server';
+
+// Subscribe to an event that calls every Minecraft tick
+system.runInterval(() => {
+	// Spams the chat with "Hello World" with world.sendMessage function in API
+	world.sendMessage("Hello World");
+
+	// or run a command in overworld dimension
+	// using native API methods (such as world.sendMessage) are recommended whenever possible.
+	world.getDimension("overworld").runCommandAsync("say Hello World");
+});
+```
+
+## Reference Documentation
+
+Official documentations on are hosted on Microsoft Learn and can be found here:
```

Comment:
Documentation is countable or uncountable; the plural forum of documentation is rarely used. I'd suggest changing to `Official documentation`